packages/client/modules/meeting/components/PokerTemplateScaleDetails.tsx

```diff
@@ -96,102 +87,67 @@ const ScaleValues = styled('div')({
 })
 
 interface Props {
-  gotoTeamTemplates: () => void
-  gotoPublicTemplates: () => void
-  viewer: PokerTemplateScaleDetails_viewer
+  team: PokerTemplateScaleDetails_team
 }
 
+const specialCards = [PokerCards.QUESTION_CARD, PokerCards.PASS_CARD] as string[]
 const PokerTemplateScaleDetails = (props: Props) => {
-  const {viewer} = props
-  const team = viewer.team!
-  const {scales} = team
-  const scale = team.scale!
-  const incomingEdges = scale.values ?? null
-  const isOwner = scale.teamId === team!.id
+  const {team} = props
+  const {id: teamId, scales, editingScaleId} = team
+  const scale = scales.find((scale) => scale.id === editingScaleId)!
+  const {values} = scale
+  const isOwner = scale.teamId === teamId
   const atmosphere = useAtmosphere()
-  const [isEditingScaleValueLabel, setIsEditingScaleValueLabel] = useState(false)
-  const [edges, setEdges] = useState([] as readonly any[])
-
-  useEffect(() => {
-    if (incomingEdges) {
-      setEdges(incomingEdges)
-      setIsEditingScaleValueLabel(false)
-    }
-  }, [incomingEdges])
-
   const gotoTemplateDetail = () => {
     commitLocalUpdate(atmosphere, (store) => {
-      store.get(team.id)?.setValue(null, 'editingScaleId')
+      store.get(teamId)?.setValue(null, 'editingScaleId')
     })
   }
   useEffect(() => gotoTemplateDetail, [])
 
-  if (edges.length === 0 && !isEditingScaleValueLabel) {
-    return <AddPokerTemplateScaleValue setIsEditing={setIsEditingScaleValueLabel} />
-  }
-
   return (
     <ScaleValueEditor>
-      <Scrollable>
-        <ScaleDetailHeader>
-          <IconButton aria-label='Back to Template' onClick={gotoTemplateDetail}>
-            <BackIcon>arrow_back</BackIcon>
-          </IconButton>
-          <ScaleDetailsTitle>{'Edit Scale'}</ScaleDetailsTitle>
-        </ScaleDetailHeader>
-        <HR />
-        <ScaleHeader>
-          <ScaleNameAndValues>
-            <EditableTemplateScaleName
-              name={scale.name}
-              scaleId={scale.id}
-              scales={scales}
-              isOwner={isOwner}
-            />
-            <ScaleValues>
-              {
-                [...scale.values.filter(({isSpecial}) => !isSpecial).map(({label}) => label), PokerCards.QUESTION_CARD, PokerCards.PASS_CARD].join(", ")
-              }
-            </ScaleValues>
-            <ScaleValues>{'Note: all scales include ? and Pass cards'}</ScaleValues>
-          </ScaleNameAndValues>
-        </ScaleHeader>
-        <TemplateScaleValueList scale={scale} />
-        <NewTemplateScaleValueLabelInput
-          isOwner={isOwner}
-          isHover={true}
-          isEditing={isEditingScaleValueLabel}
-          setIsEditing={setIsEditingScaleValueLabel}
-          scale={scale}
-        />
-        {!isEditingScaleValueLabel && scale.values.length < Threshold.MAX_POKER_TEMPLATE_SCALES &&
```

Comment:
note that `MAX_POKER_TEMPLATE_SCALES` is referring to the number of scales for a team.
what we want is the number of scale values a particular scale can have (added `MAX_POKER_SCALE_VALUES`)